u8timeseries/tests/test_timeseries.py

```diff
@@ -84,9 +84,27 @@ def test_ops(self):
         seriesA = TimeSeries(pd.Series([2 for _ in range(10)], index=self.pd_series1.index))
         targetAdd = TimeSeries(pd.Series(range(2, 12), index=self.pd_series1.index))
         targetSub = TimeSeries(pd.Series(range(-2, 8), index=self.pd_series1.index))
+        targetMul = TimeSeries(pd.Series(range(0, 20, 2), index=self.pd_series1.index))
+        targetDiv = TimeSeries(pd.Series([i / 2 for i in range(10)], index=self.pd_series1.index))
+        targetPow = TimeSeries(pd.Series([float(i ** 2) for i in range(10)], index=self.pd_series1.index))
 
         self.assertEqual(self.series1 + seriesA, targetAdd)
+        self.assertEqual(self.series1 + 2, targetAdd)
```

Comment:
Not really relevant for this PR, but for info you can also specify a message to be displayed in case a test fails. Especially useful when testing sophisticated methods